configuration/redis/redis.go

```diff
@@ -265,32 +265,34 @@ func (r *ConfigurationStore) Get(ctx context.Context, req *configuration.GetRequ
 
 func (r *ConfigurationStore) Subscribe(ctx context.Context, req *configuration.SubscribeRequest, handler configuration.UpdateHandler) (string, error) {
 	subscribeID := uuid.New().String()
+	keyStopChanMap := make(map[string]chan struct{})
 	if len(req.Keys) == 0 {
 		// subscribe all keys
 		stop := make(chan struct{})
-		r.subscribeStopChanMap.Store(subscribeID, stop)
+		keyStopChanMap[keySpaceAny] = stop
```

Comment:
Should we use synchronised map here also ?

Reply:
`keyStopChanMap` is unique for a given subscription ID, unlike `subscribeStopChanMap` which is a common map for all subscription ids. imo it should be okay for this to be normal map.

Reply:
got it. make sense.